alembic/env.py

```diff
@@ -38,8 +38,9 @@ def get_target_metadata():
 
 def get_db_url():
     setup_pythonpath()
-    from backend.util import Config as BackendConfig
-    return BackendConfig.get_config_value(BackendConfig.VALUE_DB_URL)
+    from backend.config import Config as BackendConfig
+    BackendConfig.init()
```

Comment:
The thing is, initialization is different for production and unit tests (that’s the whole point of it).

The only better thing I see is to leave explicit initialization for unit tests only (see `BaseDbAwareTest.init_database` and `TestApplyUpdate._init_database`) and normal code will just perform default initialization on first `get` for uninitialized config. It will still be quite ugly, but less ugly.

What do you think?